std/base64.d

```diff
@@ -1721,6 +1721,21 @@ template Base64Impl(char Map62th, char Map63th, char Padding = '=')
     }
 }
 
+///
+@safe unittest
+{
+    import std.string : representation;
+
+    // pre-defined: alias Base64 = Base64Impl!('+', '/');
+    ubyte[] emptyArr;
+    assert(Base64.encode(emptyArr)                == "");
```

Comment:
Let's avoid aligned equals signs. See the `Class/Struct Field Declarations` section here https://dlang.org/dstyle.html